### PR TITLE
Ability to delete multiple layers at once (Issue-2578)

### LINKED ISSUE #2578: Ability to delete multiple layman server layers at once
# Feature Request

**Describe the solution you'd like**
It would come in handy if the user could be able to delete multiple server layers at once, to clean up the database. Curretly, as far as I know, it is only possible to do it one by one.

### PR DESCRIPTION
**Pull request by DailisLangovskis**
## Description

This PR resolves the issue 2578 and also fixes the deletion of the layers, be waiting Laymans response with async/await.

Also added toast message, when layers are removed

## Related issues or pull requests

closes #2578 

## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [x] Feature
- [ ] Dependency updates
- [x] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)
- [ ] I am unsure (we'll look into it together)

## Do you introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] I am unsure (no worries, we'll find out)

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the [MIT License]
- [x] I have followed the [guidelines for contributing](https://github.com/hslayers/hslayers-ng/blob/master/CONTRIBUTING.md)
- [x] The proposed change fits to the content of the [code of conduct](https://github.com/hslayers/hslayers-ng/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)
- [ ] I'm lost; why do I have to check so many boxes? Please help!

### CODE REVIEW

**Comment by DailisLangovskis** [comment]
Will appreciate some critics and suggestions or addons

File: projects/hslayers/src/assets/locales/en.json [hunk 1]
```diff
@@ -569,7 +573,10 @@
       "2": "Unable to open raster file",
       "18": "Missing one or more Shape files",
       "50": "Error when normalizing raster file"
-    }
+    },
+    "layerSuccessfullyRemoved": "Layer successfully removed",
+    "allLaymanlayersSuccessfullyRemoved": "All Layman layers successfully removed",
+    "laymanDeleteRequest": "Layman delete layers request"
```
Comment (FilipLeitner) [inline projects/hslayers/src/assets/locales/en.json:579]: Id suggest reformulating these because user doesnt necessarily need to know whats Layman

**Comment by DailisLangovskis** [comment]
There are still missing translations for CS and SK.

**Comment by DailisLangovskis** [comment]
> Can you squash together the similar commits? I will then merge and then you can rebase the other branch (toPromise rewrite), ok?

I will do some fixing here and then will squash!

**Comment by jmacura** [comment]
Translations are on the go.

**Review by raitisbe — APPROVED** [review]

**Comment by jmacura** [comment]
obrazek

non i18n string

**Comment by jmacura** [comment]
Also incomplete sk translation @FilipLeitner 
obrazek

**Pull request merged**